cmd/homi/setup/cmd.go

```diff
@@ -127,6 +127,7 @@ var HomiFlags = []cli.Flag{
 	altsrc.NewInt64Flag(shanghaiCompatibleBlockNumberFlag),
 	altsrc.NewInt64Flag(kip103CompatibleBlockNumberFlag),
 	altsrc.NewStringFlag(kip103ContractAddressFlag),
+	altsrc.NewInt64Flag(cancunCompatibleBlockNumberFlag),
```

Comment:
It's unrelated to Cancun hardfork, but can you reorder to kip103number - kip103address - shanghai - cancun throughout the codebase?
in cmd.go:128, cmd.go:685, flags.go:481, params/config.go:285, params/config.go:300, params/config.go:353